### PR TITLE
Add ability to import markdown files

### PR DESCRIPTION
**Pull request by phillmv**
ref https://github.com/phillmv/arquivo/pull/30 cc @thepwagner

Adds the ability to import random markdown files, so that Arquivo can be abused as a static site generator.

---

This is a separate PR because I thought that by using the commandline steps in that pull request, i.e.

```
git checkout -b thepwagner-markdown-import master
git pull https://github.com/thepwagner/arquivo.git markdown-import
```

I'd be pushing to the same PR but turns out I should have pushed a separate branch and rn cannot be arsed to figure out the right git incantation.

### CODE REVIEW

**Comment by phillmv** [comment]
i may keep tweaking this, holler at me if it don't work!

**Comment by phillmv** [comment]
it occurs to me now that i have to update the docker image before i try using the default action again.

File: app/models/sync_from_disk.rb [hunk 1]
```diff
@@ -44,9 +44,25 @@ def import_and_sync!(deleted: [], changed: [])
   def import!
     raise "Path bad" unless File.exist?(notebook_path)
 
+    # if we have a notebook.yaml file, we're probably dealing with a normal
```
Comment (thepwagner) [inline app/models/sync_from_disk.rb:47]: I like this, thanks!
I look forward to deleting my (boilerplate) notebook.yaml.

**Comment by phillmv** [comment]
@thepwagner https://github.com/phillmv/public2

there's like a bug in how images work but https://phillmv.github.io/public2/ it mostly works

**Comment by phillmv** [comment]
also updated how the arquivo-export action works, should be more to your liking ;p tho maybe not necc the cascading dockerfiles in /arquivo

**Pull request merged**